packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/ForkButton/ForkButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { useSignals, useStore } from 'app/store';
+import { ProgressButton, ForkIcon } from './elements';
+
+export const ForkButton = observer(() => {
+  const {
+    editor: { forkSandboxClicked },
+  } = useSignals();
+  const {
+    editor: {
+      isForkingSandbox,
+      currentSandbox: { owned },
+    },
+  } = useStore();
+
+  return (
+    <ProgressButton
+      onClick={() => {
```

Comment:
@SaraVieira This can just be `onClick={forkSandboxClicked}`

Reply:
YES! Updated! Thank you!